docs/developers/quickstart/ethereum-differences.mdx

```diff
@@ -59,12 +59,10 @@ forks, and updates to the blockchain.
         <tr>
         <td>`PUSH0`</td>
         <td>Pushes the constant value 0 onto the stack</td>
-        <td>Pushes the constant value 0 onto the stack if correct version of Solidity compiler is 
-        configured</td>
-        <td>The PUSH0 opcode compatibility was introduced in Solidity compiler version 0.8.20, 
-        which came after the London release. However, Linea currently supports Solidity compiler 
-        version 0.8.19 and lower, which aligns with the London release of the Ethereum mainnet. 
-        To resolve this issue, recompile your contract using Solidity version 0.8.19 or lower.</td>
+        <td>n/a</td>
+        <td>Introduced in Solidity compiler version 0.8.20, which came after the London release. 
+        However, Linea currently supports Solidity compiler version 0.8.19 and lower, aligning with 
+        the London release of the Ethereum Mainnet.</td>
```

Comment:
This should technically read something like:

```
PUSH0 was introduced in the Shanghai upgrade and available in Solidity compiler version 0.8.20, which came after the London release.

However, Linea currently supports compiled code targeting the London release of the Ethereum Mainnet. Using the EVM Version of London when compiling, or using Solidity compiler 0.8.19 will align with Linea's capabilities.
```

Reply:
addressed in #650